WorkflowUI/Sources/Screen/Screen.swift

```diff
@@ -26,4 +26,10 @@
         func viewControllerDescription(environment: ViewEnvironment) -> ViewControllerDescription
     }
 
+    extension Screen {
+        public func update(viewController: UIViewController, with environment: ViewEnvironment) {
```

Comment:
- [ ] Docs

Reply:
Is this used?

It seems easy enough to just call the logic this abstracts, I'd be fine leaving this convenience out :shrug:

Reply:
It's used in Market somewhat commonly to avoid having to do the description dance which saves a line of code or so, which IMO is nice, but also fine to remove it.